<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsWidget</class>
 <widget class="QGroupBox" name="ResultsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Results</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="bottom_stress">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="top_stress_lbl">
     <property name="text">
      <string>&amp;sigma;&lt;sub&gt;top&lt;/sub&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="bottom_stress_units">
     <property name="text">
      <string>MPa</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="top_stress">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="top_stress_units">
     <property name="text">
      <string>MPa</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="bottom_stress_lbl">
     <property name="text">
      <string>&amp;sigma;&lt;sub&gt;bottom&lt;/sub&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="NA_stress_label">
     <property name="text">
      <string>&amp;sigma;&lt;sub&gt;NA&lt;/sub&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="NA_stress_units">
     <property name="text">
      <string>MPa</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="NA_stress">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>top_stress</tabstop>
  <tabstop>NA_stress</tabstop>
  <tabstop>bottom_stress</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
